### DISCUSSION TITLE
Edge browser plug-in

### DISCUSSION THREAD

**Discussion by p6002** [ArchiveBox/ArchiveBox#1135] [General]
How do I get the plug-in to work in the Edge browser? 

I've tried disabling browser security but it doesn't help. In Firefox on the same computer the plug-in works without any problems.

**Comment by pirate** [comment]
Sorry I cant provide Windows/Edge support as I don't own a Windows machine and it's already a lot to support Linux, macOS, BSD.

The extension is maintained by a contributor over at tjhorner/archivebox-exporter, maybe he has a suggestion.
